scripts/index.js

```diff
@@ -1 +1,245 @@
+
+
+// 1. create the array.
+
+arr = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
+
+// 2. create squareNumbers function and pass in the array as an argument.
+
+function squareNumbers(array) {
+
+    // 3. Go over each value, squaring each value with my square function, and mapping it into a new array.
+    
+const squares = arr.map(square => Math.sqrt(square));
+    return squares;
+}
+
+console.log(squareNumbers(arr));
+
+
+// Cities 1 - Write a function which takes an array of city objects like such:
+
+// var cities = [
+//     { name: 'Los Angeles', temperature: 60.0},
+//     { name: 'Atlanta', temperature: 52.0 },
+//     { name: 'Detroit', temperature: 48.0 },
+//     { name: 'New York', temperature: 80.0 }
+//   ];
+//   as input and returns a new array containing the cities whose temperature is cooler than 70 degrees.
+
+
+
+var cities = [
+    { name: 'Los Angeles', temperature: 60.0},
+    { name: 'Atlanta', temperature: 52.0 },
+    { name: 'Detroit', temperature: 48.0 },
+    { name: 'New York', temperature: 80.0},
+];
+
+// // 1. Write the coolCities function and pass in the 'cities' array as an argument.
+
+function coolCities(cool) {
+
+// // 2. scan through the cities array, filtering out cities with the desired temp range.
+
+return cool.filter(myObject => myObject.temperature < 70);
+
+// }
+
+console.log(coolCities(cities));
+
+// Cities 2 - Write a function which takes an array of city objects like the above problem as input and returns an array of the cities names. 
+
+
+
+var cities = [
+    { name: 'Los Angeles', temperature: 60.0},
+    { name: 'Atlanta', temperature: 52.0 },
+    { name: 'Detroit', temperature: 48.0 },
+    { name: 'New York', temperature: 80.0 },
+];
+
+// 1. Write a function which takes in an array of city objects.
+
+function cityNames(array) {
+
+// // 2. Iterate over the city values, only returning the city names. (Use map)
+    
+let names = cities.map(myObject => myObject.name)
+return names;
+
+}
+
+// // 4. call the cityNames function.
+
+console.log(cityNames(cities))
+
+
+
+3 times - Given this function: 
+
+function call3Times(fun) {
+    fun();
+    fun();
+    fun();
+}
+
+// Use the call3Times function to print "Hello, world!" 3 times
+
+function helloWorld() {
+    console.log('Hello, World!');
+}
+
+function call3Times(fun) {
+    fun();
+    fun();
+    fun();
+}
+
+call3Times(helloWorld);
+
+// Good Job! Given an array of people's names
+// print out 'Good job, {name} for each person's name, one on a line. 
+
+// 1. Create an array assigned to 'people'
+
+const people = [
+    'Dom',
+    'Lyn',
+    'Kirk',
+    'Autumn',
+    'Trista',
+    'Jesslyn',
+    'Kevin',
+    'John',
+    'Eli',
+    'Juan',
+    'Robert',
+    'Keyur',
+    'Jason',
+    'Che',
+    'Ben'
+];
+
+// 2. Create a function that goes over each value, attaching each name to the phrase, "Good job, x!"
+
+function goodJob(array) {
+const greeting = people.map(people => `Good job, ${people}!`);
```

Comment:
for clarity, i would give your element in your arrow function a different name than the array passing in. so for this one, you might do `people.map(person => 'good job, ${person}');` 
but nice use of string interpolation too!